tests/drivers/adc/adc_api/src/test_adc.c

```diff
@@ -170,7 +177,14 @@ static void check_samples(int expected_count)
 static int test_task_one_channel(void)
 {
 	int ret;
+	const struct adc_sequence_options options = {
+		.interval_us     = 10,
+		.extra_samplings = 0,
+	};
 	const struct adc_sequence sequence = {
+#if defined(CONFIG_ARC)
+		.options     = &options,
```

Comment:
This is causing CI failures on non ARC boards because `options` isn't being used.